### DISCUSSION TITLE
Work with search results

### DISCUSSION THREAD

**Discussion by Randalix** [gokcehan/lf#969] [Q&A] [answered]
WIth "regular" filebrowers e.g. windows filemanager you can search in a directory and then continue filter the search results.
Lets say I'm in folder xyz and search for foo. All files in subdirectories of xzy containig foo are listed. Now I'can e.g. sort them by date; select some files and e.g. delete them.

Is this something which is possible with lf? 
Can lf lists search results from find as if they are in the same directory? E.g. Again in directory xyz; find . -iname *foo*; have the results shown in lf?

Thanks in advance.

**Comment by CatsDeservePets** [comment, marked as answer]
You can `filter` files by name and even supply multiple filters. The results can be sorted by whatever you want. However, these filters do not show results from multiple directories.
